sway-lib-std/src/tx.sw

```diff
@@ -166,6 +166,15 @@ pub fn get_script_data<T>() -> T {
     }
 }
 
+/// Get the script bytecode
+fn get_script_bytecode<T>() -> T {
```

Comment:
Is there no need to pass in the script bytecode length in addition to the start?

Reply:
I've been handling the length implicitly by casting to a `u64` array of the correct length, so what you actually get is the script data padded to the next whole word. I will document this in the comments.

Reply:
Resolved by [357182b](https://github.com/FuelLabs/sway/pull/2142/commits/357182b7379b764ea3b79d65809bd54397d60e3d)